app/controllers/merchants_controller.rb

```diff
@@ -1,6 +1,10 @@
 class MerchantsController < ApplicationController
 
   def index
+    @merchants = User.where(role: 'merchant')
```

Comment:
Change to: `@merchants = User.where(role: 'merchant', enabled: true)`

Reply:
good catch, done!